src/DepartmentService.Broker/Consumers/CreateDepartmentPendingUserConsumer.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Threading.Tasks;
+using DigitalOffice.Models.Broker.Publishing;
+using LT.DigitalOffice.DepartmentService.Data.Interfaces;
+using LT.DigitalOffice.Kernel.RedisSupport.Helpers.Interfaces;
+using MassTransit;
+
+namespace LT.DigitalOffice.DepartmentService.Broker.Consumers
+{
+  public class CreateDepartmentPendingUserConsumer : IConsumer<ICreatePendingUserPublish>
+  {
+    private readonly IDepartmentUserRepository _departmentUserRepository;
```

Comment:
Since it's single maybe just repository?

Reply:
as for me i just like this naming more than just repository